build.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2019 Advanced Micro Devices, Inc. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
`gfxreconstruct`'s core license is already Apache 2.0, so we don't need to reiterate the license in all the included files.  (Its presence when unneeded is misleading, as the reader is led to think it' s a different license than the core license.)